middleware/expandDocument.js

```diff
@@ -0,0 +1,72 @@
+const dotProp = require('dot-prop');
+const _ = require('lodash');
+
+function service(esclient) {
+  function expandDocument(doc) {
```

Comment:
Could you please add a couple of comment lines about what `expandDocument` does?

Reply:
addd